package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "atd-product",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "private": true,
   "dependencies": {
-    "@netlify/plugin-nextjs": "^5.11.2",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/user-event": "^14.6.1",
```

Comment:
the diff is confused: i did not add these `testing-library` packages.

Reply:
I removed these in this PR https://github.com/cityofaustin/atd-product/pull/108, its weird tho that I try to merge main to this branch and it wont get rid of these packages. if we merge as is, they are going to be re-added.....

Reply:
Ah ha! In that case I probably undid those changes while resolving conflicts. Will remove 🙏